### PR TITLE
Changed the HTML file according to the concept of Accessibility

### PR DESCRIPTION
**Pull request by TapishKhari**
Changed the HTML file according to the concept of Accessibility

1. Page title : I have changed the title to "Portfolio of Tapish Khari".
2. Image alternative text : I have added the description of the image under alt tag.
3. Text heading : No issue found
4. Color Contrast : It exactly according to the template 2 in figma.
5. Interaction : If we change the settings of the browser to read the website , its working fine. Browsers like chrome, Firefox.
6. Moving Content : There was no such thing in the figma template 2.
7. Multimedia :  There was no such thing in the figma template 2.
8. Basic structure of the page : No issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,11 +38,11 @@ <h1>Hey There.
...
-                <li><a href="#"><img src="images/Github.png" alt="github-icon"></a> </li>
-                <li><a href="#"><img src="images/linkedin.png" alt="linked-icon"></a> </li>
-                <li><a href="#"><img src="images/chill.png" alt="chill-icon"></a> </li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon"></a> </li>
-                <li><a href="#"><img src="images/Medium.png" alt="Medium-icon"></a> </li>
+                <li><a href="#"><img src="images/Github.png" alt="github-icon image"></a> </li>
+                <li><a href="#"><img src="images/linkedin.png" alt="linked-icon image"></a> </li>
+                <li><a href="#"><img src="images/chill.png" alt="chill-icon image"></a> </li>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon image"></a> </li>
+                <li><a href="#"><img src="images/Medium.png" alt="Medium-icon image"></a> </li>
```
Comment (awais-amjed) [inline index.html]: Kinldy remove the `image` word from the alt texts as described [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images) 😃

image

File: index.html [hunk 2]
```diff
@@ -53,7 +53,7 @@ <h2 class="myworks">My Recent Works</h2>
                         <hr class="line">
                     </div>
                     <div class="image-desktop">
-                        <img src="images/Img_Placeholder.png" alt="images" class="image">
+                        <img src="images/Img_Placeholder.png" alt="images of recent works Dashboard" class="image">
```
Comment (awais-amjed) [inline index.html]: Kindly remove `images of` from alt text as described above 🤗

File: index.html [hunk 3]
```diff
@@ -190,7 +190,7 @@ <h3 class="about-me">About me</h3>
             <section class="icons-desktop">
                 <div class="language-about">
                     <div class="Langua">
-                        <img src="images/Rectangle-55.png" alt="image">
+                        <img src="images/Rectangle-55.png" alt="image of square icon">
```
Comment (awais-amjed) [inline index.html]: Kindly remove `images of` from alt text as described above 🤗

File: index.html [hunk 4]
```diff
@@ -202,7 +202,7 @@ <h3 class="lan">Languages</h3>
                 </div>
                 <div class="frame-cont">
                     <div class="frameworks">
-                        <img src="images/Rectangle-56.png" alt="image">
+                        <img src="images/Rectangle-56.png" alt="image of square icon">
```
Comment (awais-amjed) [inline index.html]: Kindly remove `images of` from alt text as described above 🤗

File: index.html [hunk 5]
```diff
@@ -215,7 +215,7 @@ <h3 class="frame">frameworks</h3>
                 </div>
                 <div class="skill-cont">
                     <div class="skills">
-                        <img src="images/Rectangle-57.png" alt="image">
+                        <img src="images/Rectangle-57.png" alt="image of circular icon">
```
Comment (awais-amjed) [inline index.html]: Kindly remove `images of` from alt text as described above 🤗

File: index.html [hunk 6]
```diff
@@ -259,11 +259,11 @@ <h3 class="Skill">Skills</h3>
...
-                        <li><a href="#"><img src="images/Github-black.png" alt="github-icon"></a> </li>
-                        <li><a href="#"><img src="images/linkedin-black.png" alt="linked-icon"></a> </li>
-                        <li><a href="#"><img src="images/chill-black.png" alt="chill-icon"></a> </li>
-                        <li><a href="#"><img src="images/twitter-black.png" alt="twitter-icon"></a> </li>
-                        <li><a href="#"><img src="images/Medium-black.png" alt="Medium-icon"></a> </li>
+                        <li><a href="#"><img src="images/Github-black.png" alt="github-icon image"></a> </li>
+                        <li><a href="#"><img src="images/linkedin-black.png" alt="linked-icon image"></a> </li>
+                        <li><a href="#"><img src="images/chill-black.png" alt="chill-icon image"></a> </li>
+                        <li><a href="#"><img src="images/twitter-black.png" alt="twitter-icon image"></a> </li>
+                        <li><a href="#"><img src="images/Medium-black.png" alt="Medium-icon image"></a> </li>
```
Comment (awais-amjed) [inline index.html]: Kindly remove `image` word from alt text as described above 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @TapishKhari , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [ ] The text of `Get in touch` button in your Contact form becomes invisible when you hover on it as shown in the aditus report below. Kindly fix that 🤗
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @TapishKhari ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Fixed Alts✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**